transcribe/scribe_modules/nvidia_smi.py

```diff
@@ -0,0 +1,27 @@
+from . import ScribeModuleBaseClass
+
+import sys
+
+
+from . lib.util import validate_length
+
+import json
```

Comment:
i don't think we are using json anywhere.

Reply:
removed json